### ISSUE TITLE
Allow a connection to a group that contains only one target

### ISSUE THREAD

**Issue by Madd0g** [ts-graphviz/ts-graphviz#1147]
## Describe the bug

I get an error for something like this:

```js

// when I use fromDot()

C1 -> {F1, F2}; // fine
C2 -> {Q3}; // error, expecting more than one target in a group
C2 -> Q3; // works

```

I used d3-graphviz to visualize and it had no problem with the single item in a group.

## To Reproduce

```typescript
const graph = `digraph G {
    node [shape=box, style=rounded];

    P [label="P"];
    
    C [shape=hexagon, style=filled, fillcolor=lightgreen];
    Q [shape=hexagon, style=filled, fillcolor=lightcoral];

    // Element nodes
    C1 [label="C1"];
    C2 [label="C2"];
    C3 [label="C3"];

    Q1 [label="Q1"];
    Q2 [label="Q2"];
    Q3 [label="Q3"];

    P -> {C, Q} [style=dotted];

    C -> {C1, C2, C3};
    Q -> {Q1, Q2, Q3};

    C1 -> {Q2, Q3};
    // this causes the error
    C2 -> {Q3};
    C3 -> {Q1, Q3};

    // Invisible edges for layout
    {rank=source; C Q}
    {rank=same; C1 C2 C3}
    {rank=same; Q1 Q2 Q3}
}`;

fromDot(graph);
// Error: EdgeTargets must have at least 2 elements
```

## Expected behavior

No error, silently handling this case.

**Comment by kamiazya** [comment]
@all-contributors please add @Madd0g  for bug.

**Comment by kamiazya** [comment]
@Madd0g 
Thanks for the reporting bug!

This bug has been fixed in the `@ts-graphviz/common@2.1.3` and `ts-graphviz@2.1.3` releases just a few minutes ago.

https://github.com/ts-graphviz/ts-graphviz/releases/tag/%40ts-graphviz%2Fcommon%402.1.3
